src/full.test.ts

```diff
@@ -6,8 +6,9 @@ import * as v1alpha from "./v1alpha1";
 describe("a check following a write of schema and relationships", () => {
   it("should succeed", (done) => {
     // Write the schema.
-    const alphaClient = v1alpha.NewClient("fulltest-sometoken", "localhost:50051", ClientSecurity.INSECURE_LOCALHOST_ALLOWED);
-    const v1client = v1.NewClient("fulltest-sometoken", "localhost:50051", ClientSecurity.INSECURE_LOCALHOST_ALLOWED);
+    const token = `fulltest-sometoken-${Math.floor(Math.random() * 1000)}`
```

Comment:
Are the random suffixes mainly to accommodate the additional clients in the new test files?

Reply:
When testing locally multiple times, tests can leave data behind and cause subsequent errors. Personally, to ensure idempotent runs, I think each test should really generate a random key every time. Is there a preference for it to be static?

Reply:
Not a preference necessarily. The serve-testing mode creates an isolated spicedb for each unique token so we currently have a static, unique token per test. The test script also runs a spicedb container and tears it down per test suite run, so a test doesn't interfere with others. We avoided random tokens initially because they weren't necessary and there's the off-chance at having a collision.

All that to say, as long as each test's token has a descriptive unique prefix, it's ok. This would support a dev keeping a spicedb instance running and run the suite multiple times. I think there are a few tokens that were copy/pasted so their prefixes need to be updated so i'll comment on those. May be worthwhile to create a helper function to generate descriptive tokens.